### PR TITLE
Patch

### PR DESCRIPTION
**Pull request by kimegede**
- Corrected App::uses() - used `\` instead of `/`
- Indent fixes
- Readme update (Load plugin)

### CODE REVIEW

**Comment by mamchenkov** [comment]
Hi @kimegede ,  thanks for the patch!  I'll check it now.

File: Console/Command/GraphShell.php [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?php
 
-App::uses('AppShell', 'Console\Command');
+App::uses('AppShell', 'Console/Command');
```
Comment (mamchenkov) [inline Console/Command/GraphShell.php]: @kimegede maybe it's better to use DS or DIRECTORY_SEPARATOR constant here, to make sure it works both on Windows and Linux/Mac.

Comment (kimegede) [inline Console/Command/GraphShell.php]: This isn't needed here.

The `App` class takes care of the DIRECTORY_SEPARATOR.
https://github.com/cakephp/cakephp/blob/2.6.0/lib/Cake/Core/App.php#L521
https://github.com/cakephp/cakephp/blob/2.6.0/lib/Cake/Core/App.php#L534-L572

Comment (dereuromark) [inline Console/Command/GraphShell.php]: I agree, that change is bad.
`App::uses('AppShell', 'Console/Command');` is correct and has always been that way :)

**Comment by dereuromark** [comment]
:+1:

File: Console/Command/GraphShell.php (hunk 1 above)
Comment (kimegede) [inline Console/Command/GraphShell.php]: @dereuromark Thanks buddy for your input!

**Comment by mamchenkov** [comment]
Thanks!

**Comment by kimegede** [comment]
Thank you as well! ;)

**Pull request merged**